Dockerfile

```diff
@@ -25,6 +26,21 @@ RUN \
 
 ARG PYTHON_MATTER_SERVER
 
+ENV chip_example_url "https://github.com/agners/matter-linux-example-apps/releases/download/v1.3.0.0"
```

Comment:
did you manage to move it to the libs org ?

Reply:
Moved: https://github.com/home-assistant-libs/matter-linux-ota-provider